src/constants/abis/wspoa.json

```diff
@@ -0,0 +1,153 @@
+[
+  {
+    "type": "function",
+    "stateMutability": "view",
+    "payable": false,
+    "outputs": [{ "type": "string", "name": "" }],
+    "name": "name",
+    "inputs": [],
+    "constant": true
+  },
+  {
+    "type": "function",
+    "stateMutability": "nonpayable",
+    "payable": false,
+    "outputs": [{ "type": "bool", "name": "" }],
+    "name": "approve",
+    "inputs": [
+      { "type": "address", "name": "guy" },
+      { "type": "uint256", "name": "wad" }
```

Comment:
What is "guy" and "wad"?

Reply:
Pretty sure `guy` is just the address that you are allowing to spend `wad` amount of your `WSPOA`. A `wad` is just an integer representation of a number with 18 decimals (so wei with the added info that it refers to a 18-decimal number).